pkg/services/sqlstore/team.go

```diff
@@ -387,8 +387,18 @@ func GetTeamMembers(query *models.GetTeamMembersQuery) error {
 	if query.External {
 		sess.Where("team_member.external=?", dialect.BooleanStr(true))
 	}
-	sess.Cols("team_member.org_id", "team_member.team_id", "team_member.user_id", "user.email", "user.login", "team_member.external", "team_member.permission", "user_auth.auth_module")
-	sess.Asc("user.login", "user.email")
+	sess.Cols(
+		"team_member.org_id",
+		"team_member.team_id",
+		"team_member.user_id",
+		"user.email",
+		"user.name",
+		"user.login",
+		"team_member.external",
+		"team_member.permission",
+		"user_auth.auth_module",
+	)
+	sess.Asc("user.email", "user.login")
```

Comment:
Sorry for not being clear: sort first by login and after by email, as it was before:
`sess.Asc("user.login", "user.email")`

Reply:
@papagian Thank you for all the back and forth and guiding me through the pull request!

:heart: